.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "cSpell.words": ["ATSDR", "CFPB", "ISSO", "cybersecurity"],
```

Comment:
Do we want these vscode files checked into source control ?

Reply:
I'm not sure where this particular list came from - I can look into this - in general though these enable local development and testing and the Azure extension in VSCode intentionally does not exclude them from the built product. I can add a `README` about how to run these locally, and meanwhile I think I can just remove this particular line, which I think was added here spuriously. The rest of this file is important for local development

Reply:
I would be as well